src/cljs/main/broadfcui/page/workspace/method_configs/method_config_editor.cljs

```diff
@@ -1,22 +1,22 @@
 (ns broadfcui.page.workspace.method-configs.method-config-editor
   (:require
-   [dmohs.react :as react]
-   [clojure.string :as string]
-   [broadfcui.common :as common]
-   [broadfcui.common.components :as comps]
-   [broadfcui.common.method.config-io :refer [IOTables]]
-   [broadfcui.common.style :as style]
-   [broadfcui.components.buttons :as buttons]
-   [broadfcui.components.entity-details :refer [EntityDetails]]
-   [broadfcui.components.sticky :refer [Sticky]]
-   [broadfcui.endpoints :as endpoints]
-   [broadfcui.page.workspace.method-configs.delete-config :as delete]
-   [broadfcui.page.workspace.method-configs.launch-analysis :as launch]
-   [broadfcui.page.workspace.method-configs.publish :as publish]
-   [broadfcui.page.workspace.method-configs.synchronize :as mc-sync]
-   [broadfcui.page.workspace.workspace-common :as ws-common]
-   [broadfcui.utils :as utils]
-   ))
+    [dmohs.react :as react]
+    [clojure.string :as string]
+    [broadfcui.common :as common]
+    [broadfcui.common.components :as comps]
+    [broadfcui.common.method.config-io :refer [IOTables]]
+    [broadfcui.common.style :as style]
+    [broadfcui.components.buttons :as buttons]
+    [broadfcui.components.entity-details :refer [EntityDetails]]
+    [broadfcui.components.sticky :refer [Sticky]]
+    [broadfcui.endpoints :as endpoints]
+    [broadfcui.page.workspace.method-configs.delete-config :as delete]
+    [broadfcui.page.workspace.method-configs.launch-analysis :as launch]
+    [broadfcui.page.workspace.method-configs.publish :as publish]
+    [broadfcui.page.workspace.method-configs.synchronize :as mc-sync]
+    [broadfcui.page.workspace.workspace-common :as ws-common]
+    [broadfcui.utils :as utils]
+    [broadfcui.common.input :as input]))
```

Comment:
The `))` should be on its own line for ease of sorting. 
Also, imports should be alphabetized, so `broadfcui.common.input` should go under `broadfcui.common.components`

Reply:
Also, the space changes look suspicious to me. Matt or Isaac should be able to help you get intellij to do the spacing correctly.

Reply:
See CONTRIBUTING.md for import formatting, for the spaces make sure you turn on one space list indent in intellij's clojure code style prefs, then reindent all the lines you've touched.